research/object_detection/g3doc/using_your_own_dataset.md

```diff
@@ -204,6 +204,6 @@ which can then be used in the config file as below.
 
 ```bash
 tf_record_input_reader {
-  input_path: "/path/to/train_dataset.record-?????-of-00010"
```

Comment:
I think this is actually needed. Instead you should update the file list above.

Reply:
You are correct, thanks for your sharp attention and catching my mistake.